src/main/features/testing-support/views/create-draft.njk

```diff
@@ -0,0 +1,21 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import linkButton, csrfProtection, errorSummary %}
+{% from "date.njk" import dateInput %}
+
+{% set title = 'Create Draft | Testing support' %}
+{% set heading = t('Create Draft') %}
```

Comment:
Personally i don't like having the same content for the header and the page. If anyone has a better suggestion, please comment

Reply:
👍